cloud/awsprovider.go

```diff
@@ -76,6 +76,8 @@ type AWSProductAttributes struct {
 	UsageType       string `json:"usagetype"`
 	OperatingSystem string `json:"operatingSystem"`
 	PreInstalledSw  string `json:"preInstalledSw"`
+	InstanceFamily  string `json:"instanceFamily"`
+	GPU             string `json:gpu`
```

Comment:
It would be outstanding if this carried some godoc, since `GPU` of type `string` could be **anything**

Same thing fro its `GPU string` friend in the GCP version below

Reply:
Done